images/cross-cloud/suse-manager/server-ondemand/content.yaml

```diff
@@ -0,0 +1,86 @@
+image:
+  profiles:
+    profile:
+      - _attributes:
+          name: Azure-llc
+          description: Azure configuration for Non-EU/Non-UK (LLC) offers
+      - _attributes:
+          name: Azure-ltd
+          description: Azure configuration for EU/UK (Ltd) offers
+      - _attributes:
+          name: EC2-llc
+          description: EC2 configuration for Non-EU/Non-UK (LLC) offers
+      - _attributes:
+          name: EC2-ltd
+          description: EC2 configuration for EU/UK (Ltd) offers
+  preferences:
+    - _include:
+        - base/common
+    - _attributes:
+        profiles: [Azure-llc,Azure-ltd]
+      _include:
+        - csp/azure/settings/suma-server
+    - _attributes:
+        profiles: [EC2-llc,EC2-ltd]
+      _include:
+        - csp/ec2/settings/suma-server
+  packages:
+    - _attributes:
+       type: bootstrap
+      _include:
+        - base/bootstrap
+    - _attributes:
+        type: image
+      _include:
+        - base/common
+        - base/sle
+        - products/suse-manager/server/ondemand
+    - _attributes:
+        type: image
+        profiles: [Azure-llc,Azure-ltd]
+      _include:
+        - csp/azure/settings/suma-server
+      archive:
+        - _attributes:
+            name: azure.tar.gz
+    - _attributes:
+        type: image
+        profiles: [EC2-llc,EC2-ltd]
+      _include:
+        - csp/ec2
+      archive:
+        - _attributes:
+            name: ec2.tar.gz
+    - _attributes:
+        type: image
+        profiles: [Azure-llc,EC2-llc]
+      package:
+        - suma-amazon-adapter-config-llc
```

Comment:
The package `suma-amazon-adapter-config-llc` is only to be included in the `EC2-llc` profile. For Azure, the equivalent package is named  `suma-azure-adapter-config-llc` and should be put into a separate `Azure-llc` profile section. Same applies for the `ltd` version of those packages. Although the Azure packages are available in Devel:PubCloud:Azure they're still empty as we need to properly setup the offer in Azure first.

From my pov this should look like:

```
    - _attributes:
        type: image
        profiles: [EC2-llc]
      package:
        - suma-amazon-adapter-config-llc
    - _attributes:
        type: image
        profiles: [EC2-ltd]
      package:
        - suma-amazon-adapter-config-ltd
    - _attributes:
        type: image
        profiles: [Azure-llc]
      package:
        - suma-azure-adapter-config-llc
    - _attributes:
        type: image
        profiles: [Azure-ltd]
      package:
        - suma-azure-adapter-config-ltd
```